iml-wire-types/src/lib.rs

```diff
@@ -1394,10 +1394,10 @@ pub enum ActiveState {
 pub enum RunState {
```

Comment:
This enum implements `Ord`, Activating needs to be where it was previously.

Reply:
I think I had it in my patch and it was added to another patch which was merged into master so when I rebased it probably moved it to the bottom where I originally put it. I'll update.